src/components/navigation/GlobalNavigation/GlobalNavigation.stories.tsx

```diff
@@ -257,7 +260,7 @@ const mpManagement: IGlobalNavigationItem[] = [
     children: [
       { hrefOptions: { href: '/' }, label: 'Platform Settings' },
       { hrefOptions: { href: '/' }, label: 'User Settings' },
-      { hrefOptions: { href: '/' }, label: <Button type="primary">Switch to Legacy UI</Button> },
+      // { hrefOptions: { href: '/' }, label: <Button type="primary">Switch to Legacy UI</Button> },
```

Comment:
Let's keep that as an example?

```suggestion
      { hrefOptions: { href: '/' }, label: <Button type="primary">Switch to Legacy UI</Button> },
```